lib/matplotlib/text.py

```diff
@@ -309,6 +316,21 @@ def get_rotation_mode(self):
         """Return the text rotation mode."""
         return self._rotation_mode
 
+    def set_antialiased(self, b):
+        """
+        Set whether to use antialiased rendering.
+
+        Parameters
+        ----------
+        b : bool or None
```

Comment:
we should either cast to bool _or_ interpret `None` as "reset to default".  Letting the `None` leak through seems like the worst of our options here.

Reply:
Thanks for your suggestion. I will make it interpret `None` as "reset to default".